Formula/offlineimap.rb

```diff
@@ -24,15 +24,18 @@ def caveats; <<-EOS.undent
     EOS
   end
 
-  test do
-    system bin/"offlineimap", "--version"
-  end
+  plist_options :startup => true
```

Comment:
Yeh, that's the best docs we have for now (but we should improve them). In this case it doesn't need `:startup` and should specify a `:manual` on how to run this outside `launchd`